prime-router/src/main/kotlin/cli/TestReportStream.kt

```diff
@@ -1561,4 +1562,53 @@ class SantaClaus : CoolTest() {
     private fun createBad(message: String): Boolean {
         return bad("***$name Test FAILED***: $message")
     }
+}
+
+class OtcProctored : CoolTest() {
+    override val name = "otcproctored"
+    override val description = "Verify that otc/proctored flags are working as expected on api response"
+    override val status = TestStatus.SMOKE
+    val failures = mutableListOf<String>()
+
+    override fun run(environment: ReportStreamEnv, options: CoolTestOptions): Boolean {
+        val otcMap = mapOf(
+            "BinaxNOW COVID-19 Antigen Self Test_Abbott Diagnostics Scarborough, Inc." to "OTC_PROCTORED_YYY",
+            "QuickVue At-Home COVID-19 Test_Quidel Corporation" to "OTC_PROCTORED_NYY",
+            "00810055970001" to "OTC_PROCTORED_NUNKUNK"
+        )
+        for ((device_id, receiver) in otcMap) {
+            ugly("Starting Otc Test: submitting a file containing a device_id: $device_id should match receiver $receiver.")
+            val reFile = FileUtilities.replaceText(
+                "./src/test/csv_test_files/input/otc-template.csv",
+                "replaceMe",
+                "$device_id"
+            )
+
+            if (!reFile.exists()) {
+                error("Unable to find file ${reFile.absolutePath} to do otc test")
+            }
+            val (responseCode, json) = HttpUtilities.postReportFile(
+                environment,
+                reFile,
+                org.name,
+                watersSender,
+                options.key
+            )
+            echo("Response to POST: $responseCode")
```

Comment:
@carlosfelix2 wrote a useful utility in his latest PR called examineResponse(...), that checks a bunch of standard stuff coming back from the POST.  Its not always appropriate, depending on the test, but you should use it whenever it is appropriate - which might be true here.

Reply:
I'll make this changes as part of next PR